### PR TITLE
Ticket 3783 - Updated to allow for Newtons as units

### PR DESCRIPTION
**Pull request by ghost**
Extended the DbUnitChecker to allow for Newtons as a unit. Added tests to the test_units.db.

### CODE REVIEW

File: test/test_units.db [hunk 1]
```diff
@@ -133,3 +133,15 @@ record(ao, "SHOULDPASS:CDEG_OVER_SS")
 	field(DESC, "Used by GEMORC because cdeg/s^2 too long")
 	field(EGU, "cdeg/ss")
 }
+
+record(ao, "SHOULDPASS:NEWTONS:N")
+{
+    field(DESC, "Newtons should be a valid EGU")
+    field(EGU, "N"
```
Comment (Tom-Willemsen) [inline test/test_units.db]: This is broken syntax, so it probably shouldn't pass...

File: test/test_units.db [hunk 2]
```diff
@@ -133,3 +133,15 @@ record(ao, "SHOULDPASS:CDEG_OVER_SS")
...
+}
+
+record(ao, "SHOULDPASS:MILINEWTONS")
+{
+    field(DESC, "Should be able to prefix Newtons")
+    field(EGU, "mN"
```
Comment (Tom-Willemsen) [inline test/test_units.db]: Also broken syntax

File: test/test_units.db [hunk 3]
```diff
@@ -133,3 +133,15 @@ record(ao, "SHOULDPASS:CDEG_OVER_SS")
...
+}
```
Comment (Tom-Willemsen) [inline test/test_units.db]: newline at EOF

Comment (ghost) [reply test/test_units.db]: Added newline.

File: test/test_units.db (hunk 2 above)
Comment (ghost) [reply test/test_units.db]: Fixed

File: test/test_units.db (hunk 1 above)
Comment (ghost) [reply test/test_units.db]: Fixed

**Review by Tom-Willemsen — APPROVED** [review]

**Pull request merged**